### DISCUSSION TITLE
How to install a custom build of onnxruntime on Windows?

### DISCUSSION THREAD

**Discussion by ae137** [microsoft/onnxruntime#11765] [General]
I am trying to create a custom build of onnxruntime for C++ with CUDA execution provider and install it on Windows (the encountered issue is independent of the custom build). I have built the library as described [here](https://onnxruntime.ai/docs/build/eps.html#cuda), using the tag `v1.10.0`, CUDA 11.4, CMake 3.21.1 and MSVC 19.29. Building and testing works fine. 

However, when I switch to `build/Windows/Release` and type `cmake --install .`, the installation folder seems to be incomplete: There is a folder `include/onnxruntime/core/providers/cpu`, but none for `cuda`. In the source directory, there is a corresponding folder `onnxruntime/core/providers`, and I wonder which of its content should end up in the installation directory?

I have found a somewhat similar discussion for Linux [here](https://github.com/microsoft/onnxruntime/issues/4725), but there are no direct equivalents for the scripts on Windows. Is it possible to bundle the installation files with CMake, or required to use Nuget?

**Comment by ytaous** [comment]
https://github.com/microsoft/onnxruntime/issues/11807
